binaries/scoot-snapshot-db/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/scootdev/scoot/os/temp"
+	"github.com/scootdev/scoot/snapshot"
+	"github.com/scootdev/scoot/snapshot/cli"
+	"github.com/scootdev/scoot/snapshot/git/gitdb"
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+func main() {
+	inj := &injector{}
+	cmd := cli.MakeDBCLI(inj)
+	if err := cmd.Execute(); err != nil {
+		log.Fatal(err)
+	}
+}
+
+type injector struct{}
+
+func (i *injector) RegisterFlags(rootCmd *cobra.Command) {}
+
+func (i *injector) Inject() (snapshot.DB, error) {
```

Comment:
Why do we need an injector? We may want to delay db construction but we can't we instead pass the necessary params into MakeDBCLI()?